yarn.lock

```diff
@@ -42784,17 +42784,17 @@ resolve@^2.0.0-next.3:
   languageName: node
   linkType: hard
 
-"supabase@npm:^1.136.3":
-  version: 1.141.0
```

Comment:
This was the version being used by CI and overnight batch job. Hopefully updating this one will fix it. Works locally but we won't know about the nightly job until code is merged.